common/roschannels/geojsonchannel.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "roschannel.hpp"
+#include "foxglove_msgs/msg/geo_json.hpp"
+
+namespace ROSChannels {
+    namespace GeoJSON {
```

Comment:
Since geoJSON can contain more than just paths, it might be good to rename this
```suggestion
    namespace GNSSPath {
```

Reply:
> Will await merge of the previous PR before reviewing fully

yes forgot to mention, this is based off of #524